BlazarData/src/test/java/com/hubspot/blazar/data/service/BranchServiceTest.java

```diff
@@ -33,18 +33,20 @@ public void testUpsertBasic() throws Exception {
     assertThat(retrieved.isPresent()).isTrue();
     assertThat(retrieved.get()).isEqualTo(inserted);
     assertThat(retrieved.get().getCreatedTimestamp()).isBetween(before, System.currentTimeMillis());
-    assertThat(retrieved.get().getUpdatedTimestamp()).isBetween(before, System.currentTimeMillis());
-    assertThat(retrieved.get().getUpdatedTimestamp()).isEqualTo(retrieved.get().getCreatedTimestamp());
+
```

Comment:
BTW in the above `assertThat(retrieved.get()).isEqualTo(inserted)` assertion you rely on an implementation of equals for GitInfo which is problematic in my opinion. GitInfo is modeling a db entry for a branch adding properties for when this db entry was created or updated. Why then the timestamps are ignored in the implementation of equals. Is it because GitInfo is essentially capturing the info for a branch and we don't care about the db entry specific info in that case? But then why equals is counting the id (surrogate key) that the db entry is getting. 
If you wanted to keep `equals` as it is now it would be better to introduce a BranchInfo class and a separate BranchInfoDbEntry class that adds the surrogate key and timestamp properties.